amy/settings.py

```diff
@@ -338,6 +338,17 @@
     },
 }
 
+# settings for PyData application
+ENABLE_PYDATA = os.environ.get('AMY_ENABLE_PYDATA', False)
```

Comment:
What are valid values for `AMY_ENABLE_PYDATA` envvar? Which ones are truthy and which ones are falsy? What if somebody sets `AMY_ENABLE_PYDATA=false`?